dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferMethods.cs

```diff
@@ -45,14 +38,7 @@ public class TransferMethods
                 S3Directory = s3Path,
             });
 
-            if (Directory.Exists(localPath))
-            {
-                return true;
-            }
-            else
-            {
-                return false;
-            }
+            return (Directory.Exists(localPath));
```

Comment:
This method returns true if the directory exists - is there a way to check the file count or otherwise make sure it actually downloaded the files?

Reply:
Checked the file count ahead of time. If the directory doesn't exist, the "old" file count will be 0 and am now comparing the file count after the operation to the beginning file count.